modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
+= Configuring PROXY protocol for an Ingress Controller
+As a cluster administrator, you can use PROXY protocol to enable communication between the Ingress Controller and an external load-balancer. An Ingress Controller is typically published with a load balancer. The connections that the Ingress Controller receives are from the load balancer, so the source address has the load balancer address, instead of the original connection address. The PROXY protocol enables a load balancer to preserve the original source addresses and include them when forwarding connections to an IngressController, which can then use the original source addresses for logging.
+
+The ability to configure `PROXY` protocol is useful when the IngressController specifies the `HostNetwork` or `NodePortService` endpoint publishing strategies, which integrate with external load-balancers.
+the cluster administrator can set the IngressController's `spec.endpointPublishingStrategy.nodePort.protocol` or `spec.endpointPublishingStrategy.hostNetwork.protocol` field to PROXY
++
+[IMPORTANT]
+====
+This feature is supported only in non-cloud deployments. For cloud deployments, use the load balancer services for automatic deployment of a cloud load balancer to target the endpoints of a service.
+====
+
+.Prerequisites
+* You created an Ingress Controller.
+* You installed the `oc` command line interface.
+
+.Procedure
+
+. Configure the Ingress Controller API to use `PROXY` protocol.
++
+.. Configure the Ingress Controller resource:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator edit ingresscontroller/default
+----
++
+.. Edit the YAML file:
++
+.Sample `clientAccess` configuration to `Global`
+[source,yaml]
+----
+  spec:
+    endpointPublishingStrategy:
+      nodePort:                 <1>
+        protocol: PROXY
+----
+<1> Instead of setting the `nodePort` field, you can also set the `hostNetwork` field.
```

Comment:
Is it ok to let users know that PROXY can be set in `hostNetwork` and `nodePort` with a callout? Or do you think I need multiple examples?